src/charm.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+
+import sys
+sys.path.append('lib')
+
+from ops.charm import CharmBase
+from ops.main import main
+from ops.model import ActiveStatus, MaintenanceStatus
+from oci_image import OCIImageResource, ResourceError
+
+
+class MultusCharm(CharmBase):
+    def __init__(self, *args):
+        super().__init__(*args)
+        self.multus_image = OCIImageResource(self, 'multus-image')
+        self.framework.observe(self.on.start, self.set_pod_spec)
```

Comment:
As mentioned above, this should change to `self.on.install`.

Reply:
done